Resources/doc/index.md

```diff
@@ -31,9 +31,9 @@ public function registerBundles()
 Generate the SSH keys :
 
 ``` bash
-$ mkdir -p app/var/jwt
-$ openssl genrsa -out app/var/jwt/private.pem -aes256 4096
-$ openssl rsa -pubout -in app/var/jwt/private.pem -out app/var/jwt/public.pem
+$ mkdir -p var/jwt # Symfony2
```

Comment:
The `var/` directory yes, but not the `jwt/` one so the `mkdir is mandatory, only the `-p` option is not.
As it will work as expected in both versions (creating the `var/` directory if not exist, otherwise going into), I would remove this comment.